app/Services/ProjectsBillingService.php

```
@@ -74,4 +76,50 @@ public function getProjectedProgressBilling($month, $year)
+    {
+        $startDate = Carbon::create($year, 1, 1)->startOfDay();
+        $endDate   = Carbon::create($year, $month, 1)->endOfMonth()->endOfDay();
+        $projects = Project::select('id', 'code', 'name', 'location', 'amount', 'ntp_date')
+            ->whereBetween('ntp_date', [$startDate, $endDate])
+            ->orderBy('ntp_date', 'asc')
+            ->get();
+        $grouped = $projects->groupBy(function ($proj) {
+            return Carbon::parse($proj->ntp_date)->format('Y-m');
+        });
+        $groupedProjects = [];
+        $monthly_net_gt = 0;
```

PHPMD (2.15.0)

The variable $monthly_net_gt is not named in camelCase. (undefined)

(CamelCaseVariableName)